### ISSUE TITLE
Unable to access permalinks settings after fresh install

### ISSUE THREAD

**Issue by zuzust** [evolution/wordpress#113]
After a fresh install of WP 4.4.2, it is not possible to access permalinks configuration from admin dashboard. It keeps on waiting and no page is loaded. Same issue on Genesis WordPress. Am I missing anything?

**Comment by EvanK** [comment]
You're still able to access the site itself, and the rest of the admin?

**Comment by EvanK** [comment]
@zuzust I am unable to replicate this, so far. Can you please verify a few things about your site structure:

* Are all your wordpress core files in `web/wp`?
* Do you have any php files in `web/` (that you did not explicitly create), other than `index.php` and `wp-config.php`?
* In your `web/.htaccess` file, is there anything outside of the `# BEGIN ... # END` blocks for Wordpress and Evolution?
* What output do you get from [running the capistrano task](https://github.com/evolution/wordpress/blob/master/docs/REF-cap-tasks.md) `wp:rewrite:list`?

**Comment by zuzust** [comment]
@EvanK,

- Yes, all wordpress core files in `web/wp`
- No, no php files under `web/` other than `index.php` and `wp-config.php`
- Nothing outside those blocks
- No able to run after vagrant box upgrade. See #114

**Comment by EvanK** [comment]
When you try to load the permalinks page in the admin, can you let it time out (rather than closing/navigating away), and then see what shows up in your apache error logs with the `evolve:logs:apache:error` [task](https://github.com/evolution/wordpress/blob/master/docs/REF-cap-tasks.md#evolvelogsapacheerror)?

**Comment by zuzust** [comment]
@EvanK, tested again, but in fact it never times out, it just waits for a response endlessly. No entries in error log, as shown in attached file. Highlitghted entries in access log correspond to access to permalink settings page.

![evolve-logs-apache-error](https://cloud.githubusercontent.com/assets/351530/13921791/edb8c688-ef79-11e5-9ef2-5ecec6b03b19.png)

![evolve-logs-apache-access](https://cloud.githubusercontent.com/assets/351530/13921795/f3dd7f2c-ef79-11e5-8e63-65808987ce96.png)

**Comment by EvanK** [comment]
That looks like it's not a page failing to load, per se, but maybe the wp admin is trying/re-trying an ajax request....

Are you seeing any javascript errors (or any messages) in the browser's console?

**Comment by zuzust** [comment]
Nope. It's insane. `options-permalink.php` is the only page that fails to load with no error logs. Everything else is ok.

![evolution](https://cloud.githubusercontent.com/assets/351530/13984023/d2e0cff6-f0f3-11e5-81de-989ed3160175.png)

**Comment by EvanK** [comment]
What versions are you using of:

* Your host os (some flavor of Ubuntu, by the look of the screenshots)
* vagrant
* virtualbox

A few more things to test...

1. Do you have any non-standard plugins or themes installed? If so, can you deactivate them all to see if the problem goes away?
2. Has wordpress made _any_ changes to your htaccess file? (You can check this by `git add`ing the htaccess _after_ site generation but _before_ installation, then `git diff`ing it after installation)
3. Regenerate the site _without_ the varnish cache (and `destroy` + re-`up` vagrant), and see if the problem persists

**Comment by zuzust** [comment]
@EvanK, I finally found what caused the permalinks issue. I use *uncomplicated firewall (ufw)* in my Ubuntu Trusty development environment. After deactivating it, the page loaded without any problem. When active the page failed to load again. I don't know which is the violating rule though.

I ran my test on a *Genesis WordPress* site (same issue there), due to #117 (hoping to see it fixed any time soon).
